_docs_integrate/version-5-migration-guide.md

```diff
@@ -21,85 +21,98 @@ required_by:
+| createdByDevice | `string`                                                                                                                                                                                                                                                            | The ID of the Device that created the RelationshipAuditLogEntry. You can use this information to track back who exactly did what. |
+| reason          | `"Creation"` \| `"AcceptanceOfCreation"` \| `"RejectionOfCreation"` \| `"RevocationOfCreation"` \| `"Termination"` \| `"ReactivationRequested"` \| `"AcceptanceOfReactivation"` \| `"RejectionOfReactivation"` \| `"RevocationOfReactivation"` \| `"Decomposition"` | The reason why the `status` of the Relationship changed.                                                                          | The values for `reason` like `"Termination"`, `"ReactivationRequested"`, `"AcceptanceOfReactivation"`, `"RejectionOfReactivation"`, `"RevocationOfReactivation"` and `"Decomposition"` have been added because of the termination of Relationships. |
+| oldStatus       | `"Pending"` \| `"Active"` \| `"Terminated"` \|`undefined`                                                                                                                                                                                                           | The `status` the Relationship had before.                                                                                         | `"Terminated"` has been added because of the termination of Relationships and therefore the `oldStatus` can have also other values than only `"Pending"` like before.                                                                               |
+| newStatus       | `"Pending"` \| `"Active"` \| `"Rejected"` \| `"Revoked"` \|`"Terminated"` \| `"DeletionProposed"`                                                                                                                                                                   | The `status` the Relationship has now.                                                                                            | `"Terminated"` and `"DeletionProposed"` have been added because of the termination of Relationships.                                                                                                                                                |
 
-The `mustBeAccepted` property of the RequestItems will be kept.
-{: .notice--info}
+For example, the first entry of the `auditLog` of the [Relationship]({% link _docs_integrate/data-model-overview.md %}#relationship) has `"Creation"` as `reason`, no `oldStatus` and `"Pending"` as `newStatus` and is created by the Identity that has created the Relationship.
 
-## New mandatory wrappers for non-standard content of Messages, RelationshipTemplates and Relationships
+### New Mandatory Wrappers for Non-Standard Content of Messages, RelationshipTemplates and Relationships
```

Comment:
If we capitalize headers we have to capitalize everything, including **for** **of** and **and**

Reply:
When using "Title Case", some words (like articles or prepositions and conjunctions of three letters and less) aren't capitalize. 

There are several stylebooks with rules (refer to https://en.wikipedia.org/wiki/Title_case), but they have in common that certain words like "and", "of" or "for" are still lowercase. There are also web tools that can convert a title into the “Title Case”, which I used to check my titles.

Reply:
crazy! good to know, thanks :)